src/lib/isTaxID.js

```diff
@@ -1226,15 +1238,26 @@ export default function isTaxID(str, locale = 'en-US') {
     if (locale in sanitizeRegexes) {
       strcopy = strcopy.replace(sanitizeRegexes[locale], '');
     }
-    if (!taxIdFormat[locale].test(strcopy)) {
-      return false;
-    }
 
-    if (locale in taxIdCheck) {
-      return taxIdCheck[locale](strcopy);
+    // Check if locale has multiple TIN
+    if (locale in multipleTinLocale) {
+      if (!localeOption) throw new Error(`Locale requires localeOption ${localeOption}`);
+
+      if (!(multipleTinLocale[locale][localeOption])) throw new Error(`Locale Option ${localeOption} not available for specified locale ${locale}`);
+
+      if (!(taxIdFormat[locale][localeOption].test(strcopy))) return false;
+    } else {
+      if (localeOption || localeOption === '') throw new Error(`Locale ${locale} do not have localeOption '${localeOption}'`);
+
+      if (!taxIdFormat[locale].test(strcopy)) {
+        return false;
+      }
+      if (locale in taxIdCheck) {
+        return taxIdCheck[locale](strcopy);
```

Comment:
Do we not need this for when a `localeOption` is properly defined?

Reply:
We would need this in future but not now, I did add this earlier but removed it  because of the code coverage.
At present we do not have any locale with multiple TIN in `taxIdCheck` object, so we won't be able to test that part of the code.

We can add this when taxIdCheck is required for a `localeOption`.

Reply:
@WikiRik  @braaar what do you think, do we proceed to add this even though there is no locale under multiple TIN which requires taxIdcheck.

Reply:
If we're not using it it is fine to not have it